src/compiler/utilities.ts

```diff
@@ -505,7 +505,7 @@ namespace ts {
         }
     }
 
-    function staticAssertNever(_: never): void {}
+    export function staticAssertNever(_: never): void {}

```

Comment:
There's a duplicate function `assertTypeIsNever` in `core.ts`, or you may prefer `Debug.assertNever` which throws.